### PR TITLE
Remove frameworkAssemblies section in project.json

### PR DESCRIPTION
**Pull request by felixwa**
Completely remove frameworkAssemblies section in project.json: this
section causes NuGet package installation failure in VS
ToolsVersion="12.0".
Bump the version to 0.2.1.

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/dev/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [x] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -19,8 +19,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Cognitive Services management functions for managing Microsoft Azure Cognitive Services accounts.")]
 
-[assembly: AssemblyVersion("0.2.0.0")]
-[assembly: AssemblyFileVersion("0.2.0.0")]
+[assembly: AssemblyVersion("0.2.1.0")]
```
Comment (markcowl) [inline src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Properties/AssemblyInfo.cs]: AssemblyVersion should remain 0.2.0.0, for backjward compatibility

**Review by markcowl — CHANGES_REQUESTED** [review]

File: src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (felixwa) [inline src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Properties/AssemblyInfo.cs]: We have publish version 0.2.0.0 in NuGet already, if I keep it as 0.2.0.0, will we be able to publish it again?

Comment (felixwa) [inline src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/Properties/AssemblyInfo.cs]: Revert AssemblyVersion and AssemblyFileVersion back  to 0.2.0.0.

File: src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/project.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.2.0-preview",
+  "version": "0.2.1-preview",
```
Comment (markcowl) [inline src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/project.json]: Almost.  In AssemblyInfo.cs, the AssemblyVersion should be 0.2.0.0 as you have specified, the AssemblyFileVersion should match the package version in packjage.json (0.2.1.0)

**Review by markcowl — CHANGES_REQUESTED** [review]

File: src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/project.json (hunk 2 above)
Comment (felixwa) [inline src/ResourceManagement/CognitiveServices/Microsoft.Azure.Management.CognitiveServices/project.json]: Alright.

**Review by markcowl — APPROVED** [review]

**Pull request merged**